### PR TITLE
Configure backend formating

### LINKED ISSUE #98: Automate Formatting - Resharper in backend

### PR DESCRIPTION
**Pull request by GodVenn**
Closes #98

### CODE REVIEW

File: backend/.config/dotnet-tools.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+  "isRoot": true,
+  "tools": {
+    "dotnet-format": {
+      "version": "5.1.250801",
+      "commands": [
+        "dotnet-format"
+      ]
+    }
+  }
+}
```
Comment (VegardOztan) [inline backend/.config/dotnet-tools.json:12]: No newline

File: backend/api.test/Services/ServiceNowServiceShould.cs [hunk 2]
```diff
@@ -24,8 +20,8 @@ public ServiceNowServiceShould()
         public void ReturnRitmFromFormJson()
         {
             //TODO: Create actual form to mock service - Waiting for service now integration
-            const string form = "{jsonForm}";
-            string ritm = _service.SubmitIpadOrderForm(form).Result;
+            const string Form = "{jsonForm}";
+            var ritm = _service.SubmitIpadOrderForm(Form).Result;
```
Comment (VegardOztan) [inline backend/api.test/Services/ServiceNowServiceShould.cs:24]: Change back to `string`

File: backend/api/Controllers/ServiceNowController.cs [hunk 3]
```diff
@@ -58,7 +54,7 @@ public async Task<ActionResult<string>> PostIpadOrderForm([FromBody] string orde
                 _logger.LogError(e, "Submitting form");
 
                 // Select appropriate status code (400 - Bad request if argument exception, 500 if something else)
-                int statusCode = e switch
+                var statusCode = e switch
```
Comment (VegardOztan) [inline backend/api/Controllers/ServiceNowController.cs:57]: Change back to `int`

Comment (GodVenn) [reply backend/api/Controllers/ServiceNowController.cs]: Done

File: backend/api.test/Services/ServiceNowServiceShould.cs (hunk 2 above)
Comment (GodVenn) [reply backend/api.test/Services/ServiceNowServiceShould.cs]: Done

**Review by VegardOztan — APPROVED** [review]

**Pull request merged**